docs/Architecture Decision Record/009-git-workflow.md

```diff
@@ -49,8 +50,9 @@ A contributor to the TDP project would always use the following steps to propose
     * removes `raft-review` label
     * adds `qasp-review` label
     * assigns the Government as the reviewer:
-        * For backend development work assign `alexsoble`, `adpennington` and `lfrohlich`
-        * For frontend work, assign the same as backend work but also include `iamjolly` and `ttran-hub` for a11y review
+        * For backend and frontend development work assign `adpennington` and `alexsoble`
+        * For frontend work submitted for review but does not yet complete the epic, `adpennington` will sign off on Raft's a11y documentation (e.g. screen captures with VoiceOver utility, summary of review, etc.) and cc: `ttran-hub`
+        * For frontend work submitted for review that completes the epic, `adpennington` will complete code review, tag `ttran-hub` + `iamjolly` via comment, and add `a11y` label to PR 
```

Comment:
One suggestion, would it make sense to mention here that Thomas and Robert will do manual review using Accessibility Insights for work that represents a completed Epic?

Reply:
Maybe not since this is just about the git flow

Reply:
we could refer to this: https://github.com/HHS/TANF-app/blob/main/docs/Technical-Documentation/how-government-will-test-a11y.md